python/nano/tutorial/notebook/training/pytorch/run-nano-howto-guides-training-pytorch-tests.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+export ANALYTICS_ZOO_ROOT=${ANALYTICS_ZOO_ROOT}
+export NANO_HOME=${ANALYTICS_ZOO_ROOT}/python/nano/src
+export NANO_HOWTO_GUIDES_TEST_DIR=${ANALYTICS_ZOO_ROOT}/python/nano/tutorial/notebook/training/pytorch
+
+TORCH_VERSION=`python -c "from bigdl.nano.pytorch.utils import TORCH_VERSION_LESS_1_12; print(TORCH_VERSION_LESS_1_12)"`
+
+set -e
+
+# disable training with native pytorch bf16 amp if torch<1.12
+if [[ $TORCH_VERSION_LESS_1_12 == True ]]
+then
+    sed -i "s/MyNano(precision='bf16').train()/#MyNano(precision='bf16').train()/" $NANO_HOWTO_GUIDES_TEST_DIR/accelerate_pytorch_training_bf16.ipynb
+fi
```

Comment:
Do we also to disable the ipex+bf16 case when torch < 1.12?

Reply:
We don't need.
But ipex bf16 requires avx512.

Reply:
Thank you! I have add check to disable bf16+ipex training when avx512 is not available.